src/olympia/devhub/templates/devhub/addons/submit/base.html

```diff
@@ -13,6 +13,12 @@
 {% endblock %}
 
 {% block content %}
+{% if submit_notification_warning %}
+    <div class="notification-box warning">
+        {# The submit notification warning is controlled by admins, HTML is allowed #}
+        {{ submit_notification_warning|safe }}
+    </div>
+{% endif %}
```

Comment:
I feel like this requires a test that at least the box is visible and can be extended to verify that html is indeed allowed. Though I didn't look into if there are any tests already covering that.